tests/gem5/cpu_tests/test.py

```diff
@@ -49,7 +49,7 @@
     for workload in workloads:
         ref_path = joinpath(getcwd(), 'ref', workload)
         verifiers = (
-                verifier.MatchStdout(ref_path),
+                verifier.MatchStats(ref_path),
```

Comment:
Just out of curiosity, does it work to have both verifiers? If so, I think we should have both.